README.md

```diff
@@ -27,6 +27,7 @@ The Bamboo.MandrillAdapter and Bamboo.SendgridAdapter **are being used in produc
 and have had no issues. It's also pretty simple to [create your own adapter]. Feel free
 to open an issue or a PR if you'd like to add a new adapter to the list.
 
+* `Bamboo.MailgunAdapter` - Ships with Bamboo. Thanks to @princemaple.
```

Comment:
I didn’t check that. I’m guessing I’ll have to link to it :) Good catch!

> On Apr 29, 2016, at 4:27 PM, Justin Kenyon <notifications@github.com> wrote:
> 
> In README.md <https://github.com/thoughtbot/bamboo/pull/140#discussion_r61639881>:
> 
> > @@ -27,6 +27,7 @@ The Bamboo.MandrillAdapter and Bamboo.SendgridAdapter **are being used in produc
> >  and have had no issues. It's also pretty simple to [create your own adapter]. Feel free
> >  to open an issue or a PR if you'd like to add a new adapter to the list.
> >  
> > +* `Bamboo.MailgunAdapter` - Ships with Bamboo. Thanks to @princemaple.
> Should you link to @princemaple's github page, or will github automatically do that?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/bamboo/pull/140/files/106b4a7277374f2306b2bb4aaddcd8e876ea2887#r61639881>